pixart.js

```diff
@@ -1 +1,45 @@
+$(document).ready(function(){
 
+  var clickcount=1;
```

Comment:
This is an interesting solution to the bonus. I'm always a bit wary of counting clicks, just because that can be a bit tricky to understand / debug. Perhaps a better way to do this would be to remove the oldest (last) swatch from the container, and then add a new one each time you set a color. This to me is conceptually a bit simpler, and probably less prone to weird bugs.